development.cfg

```diff
@@ -1,4 +1,4 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```

Comment:
I copied that from one of our other packages where @jone switched to Plone 5.1 in `development.cfg`: https://github.com/4teamwork/ftw.testbrowser/commit/2e896f1072fad633e305f8d87f45dbadd3532597

@jone, what's our policy here? Could you please answer the question of @elioschmutz?

Reply:
Fair point. We are not doing that consistently.

When adding 5.1 support to a package it is helpful being able to actually boot it up with 5.1 and verify that it works; for that the change in `development.cfg` is needed while developing 5.1 support.
But since most of our stack is still 4.3, it is not helpful later when returning to the daily work.

I caught myself switching the version in `development.cfg` back and forth, so both have it's reasons.

But for practical reasons I tend to not change to 5.1 yet (although I have in some packages, I know).